controllers/components/kserve/kserve_support.go

```diff
@@ -266,3 +267,14 @@ func ownedViaFT(cli client.Client) handler.MapFunc {
 		return []reconcile.Request{}
 	}
 }
+
+func isLegacyOwnerRef(or metav1.OwnerReference) bool {
```

Comment:
maybe bad memory, but i dont think we set ownerref pre-2.17
it is using source to indicate it is from component or dsci

Reply:
In the incubation branch so likely in a ODH release, the owner is set to the DSC:
- https://github.com/opendatahub-io/opendatahub-operator/blob/4f3efe216ce4d875c855db22ee194efbb1f61f3d/components/kserve/kserve.go#L129C15-L129C34
- https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/components/kserve/kserve_config_handler.go#L151

Reply:
you are right, it was introduced in https://github.com/opendatahub-io/opendatahub-operator/commit/10dd5543ca098e17d8265d29d3ba365867ec5b0a